application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+
+security:
+  user:
+    name: "admin"
+    password: "admin"
```

Comment:
This is the development configuration intended to be used by developers when running locally.  It is picked up when running the app from the root of the project.  This file does not get included when building the app into a JAR.